apps/pdc-frontend/src/util/openFormsValidation.ts

```diff
@@ -0,0 +1,43 @@
+'use server';
+
+import { redirect } from 'next/navigation';
+import { ErrorHandler } from './fetchData';
+import { createOpenFormsApiUrl } from './openFormsSettings';
+
+type OpenFormValidatorFunction = {
+  formId: string;
+};
+export const openFormValidator = async ({ formId }: OpenFormValidatorFunction) => {
+  if (!formId || !process.env.OPEN_FORMS_API_TOKEN) return;
```

Comment:
```suggestion
  if (!formId) return;
```

Token is niet nodig hier en volgens mij hebben we de token überhaupt niet in de frontend